scripts/go.mod

```diff
@@ -5,10 +5,10 @@ go 1.15
 require (
 	github.com/brancz/gojsontoyaml v0.0.0-20191212081931-bf2969bbd742
 	github.com/bwplotka/mdox v0.9.0
-	github.com/golangci/golangci-lint v1.44.2
```

Comment:
@simonpasquier without bumping this f.e got this https://github.com/golang/go/issues/51706

Reply:
Ack. For clarity I've submitted #4790 to bump the golangci-lint version.

Reply:
Thank you very much @simonpasquier!!